patches/chrome-BUILD.gn.patch

```diff
@@ -60,31 +60,32 @@ index 9bb34fe40f0cc08cb832ee6728ac8e32458c969b..d9996a8931038ceebb89b77db255f675
        }
  
        extra_substitutions = [
-@@ -915,6 +922,7 @@ if (is_win) {
+@@ -913,7 +920,7 @@ if (is_win) {
+     }
+ 
      if (enable_updater) {
-       if (is_chrome_branded) {
+-      if (is_chrome_branded) {
++      if (use_prebuilt_omaha4) {
          sources += [ "//third_party/updater/chrome_mac_universal_prod/cipd/${updater_product_full_name}.app" ]
-+      } else if (use_prebuilt_omaha4) { sources += [ "//build/mac_files/omaha4/${updater_product_full_name}.app" ]
```

Comment:
still seeing the same condition here.

Reply:
It uses the same variable `use_prebuilt_omaha4`, but I believe it is not the same condition.

Upstream is:

```
if (is_chrome_branded) {
  ...
} else {
  ...
}
```

The old patch added `else if (use_prebuilt_omaha4) { ... }`:

```
if (is_chrome_branded) {
  ...
} else if (use_prebuilt_omaha4) { ... 
} else {
  ...
}
```

The new patch instead replaces `is_chrome_branded` by `use_prebuilt_omaha4`:

```
if (use_prebuilt_omaha4) {
  ...
} else {
  ...
}
```

Reply:
ah, you're right. the `} else {` line is actually restored.

Reply:
We shouldn't be patching like this, see https://github.com/brave/brave-core/pull/29811/files#r2175663577